stdlib/libLLVM_jll/test/runtests.jl

```diff
@@ -3,5 +3,6 @@
 using Test, Libdl, libLLVM_jll
 
 @testset "libLLVM_jll" begin
-    @test dlsym(libLLVM_jll.libLLVM_handle, :LLVMInitializeTarget; throw_error=false) !== nothing
+    # Try to find a symbol from the C API of libLLVM as a simple sanity check.
+    @test dlsym(libLLVM_jll.libLLVM_handle, :LLVMContextCreate; throw_error=false) !== nothing
```

Comment:
@vchuravy Not sure how to make this code compatible with LLVM 15 back again. Is there some way to check the version of `libLLVM_jll`?

Reply:
Isn't `LLVMContextCreate` available in LLVM 15?  When I [blamed that function](https://github.com/llvm/llvm-project/blame/6009708b4367171ccdbf4b5905cb6a803753fe18/llvm/include/llvm-c/Core.h#L518) I found it was introduced 15 years ago.